package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "generator-react-zeal",
-  "version": "0.1.9",
+  "version": "0.2.0",
```

Comment:
Minor nit: I normally do a separate commit with a version bump and CHANGELOG update just before doing a release, rather than making it part of another commit.  This gives me the flexibility to put multiple commits in a release, and also avoids merge conflicts if I have multiple PRs all trying to bump the version number. Up to you, though.